xbmc/interfaces/builtins/SkinBuiltins.cpp

```diff
@@ -112,14 +112,18 @@ static int SelectBool(const std::vector<std::string>& params)
 
   CGUIDialogSelect* pDlgSelect = (CGUIDialogSelect*)g_windowManager.GetWindow(WINDOW_DIALOG_SELECT);
   pDlgSelect->Reset();
-  pDlgSelect->SetHeading(CVariant{g_localizeStrings.Get(424)});
+  int iHeader = atoi(params[0].c_str());
```

Comment:
and drop `iHeader` and use atoi in `g_localizeStrings.Get(atoi(params[0].c_str()))` like we did within the loop.